### ISSUE TITLE
cd depends no such file or directory

### ISSUE THREAD

**Issue by dollardjm** [bitcoin/bitcoin#10474]
When running is the bash shell when I enter cd depends I keep getting "no such file or directory" returned.   Any suggestions on what I am doing wrong?

**Comment by jonasschnelli** [comment]
Can you give us some more details?
The `depends/` directory should be at git root level.
Maybe do a fresh checkout or a git clean (`git clean -dfx` <-- DANGER)

**Comment by dollardjm** [comment]
Jonas,


Attached is a screenshot of where I am getting stuck with the BIP 148 node install.  I have gone back through the instructions on Github, but I keep ending up in the same spot.


Not sure what I am doing wrong exactly and any thoughts you may have are appreciated.


BTW I really enjoyed the podcast episode you were on with Trace Mayer explaining hierarchical deterministic keys.


Thanks!!


-Jon


Sent from Outlook<http://aka.ms/weboutlook>


________________________________

**Comment by MarcoFalke** [comment]
Make sure that your pwd is in the git repo. (You could call `git log -1` to verify this)

Then change the directory to the one printed by `git rev-parse --show-toplevel`.

Finally, `cd depends` should work. Otherwise, please share the output of the above commands.

**Comment by dollardjm** [comment]
Marco,


Attached is a screen shot after trying the git log -1 command and the error I receive.  I also tried looking at the root folder and got a an error that permission is denied.


-Jon


Sent from Outlook<http://aka.ms/weboutlook>


________________________________

**Comment by MarcoFalke** [comment]
You'd need to upload the screenshot directly on GitHub (https://github.com/bitcoin/bitcoin/issues/10474#issuecomment-307635876). Or even better, just copy paste the text from the terminal.

On a general note, this is not a support channel. Support requests that are not issues with our code, are best directed to the [Bitcoin StackExchange](https://bitcoin.stackexchange.com).

**Comment by laanwj** [comment]
This is a basic UNIX file system navigation issue, not bitcoin related. Would make sense to read up on using the shell.
Anyhow can you check with `pwd` directory you are in, or with `ls` if there is a directory `depends` there. If not, `cd` won't work.

**Comment by dollardjm** [comment]
Thanks Marco for the response.  I wasn't sure what the right channel was for support so I thought I would start on Github first...I will check out the bitcoin stack exchange and see if anyone else has had a similar issue and can help me get this figured out.  What I don't understand is that I am following the instructions from Github for Windows 10 exactly and am running into problems.  I am sure I am not doing something right, but I am just not sure what yet.


Thanks again for your help!


-Jon


Sent from Outlook<http://aka.ms/weboutlook>


________________________________

**Comment by dollardjm** [comment]
Thanks Wladimir for the feedback.  I am following the Github instructions for a Windows 10 installation exactly as written.  It looks like the unix commands run without an issue in the shell.  But when I go to find the depends directory it says its not there.


Marco suggested I checkout the bitcoin stack exchange to see if there are any hints over there as to what I am doing wrong.  I am sure it isn't bitcoin related too but something up with linux running on Windows 10.  I am just not sure what...yet.


-Jon


Sent from Outlook<http://aka.ms/weboutlook>


________________________________

**Comment by MeshCollider** [comment]
This should probably be closed, as @laanwj said, its not an issue with bitcoin

**Comment by MarcoFalke** [comment]
If you don't provide the output of `pwd`, `ls` et al. there is nothing we can do.

**Comment by systemeintegratedsolutions** [comment]
The issue is perhaps that the OP did not clone the repo. I made the same mistake. The tutorial wasn't very clear that one should be in the repo directory before changing to the 'depends' folder. I first thought it was perhaps a directory in the bash somewhere but after not being able to find it I found this post and then clicked that the directory is in the source. After that I continued fine with the tutorial but hit another snag which I will post as separate issue.
